pkg/kubelet/pleg/generic.go

```diff
@@ -275,6 +288,15 @@ func (g *GenericPLEG) relist() {
 				metrics.PLEGDiscardEvents.Inc()
 				klog.ErrorS(nil, "Event channel is full, discard this relist() cycle event")
 			}
+			// Log exit code of containers when they finished in a particular event
+			if events[i].Type == ContainerDied {
+				containerID, ok := events[i].Data.(string)
```

Comment:
nit: you can probably inline this to avoid nested if:

```
if containerID, ok := events[i].Data.(string); ok {
   // ...
}

Reply:
Updated. Thanks